es/ae/aens.js

```diff
@@ -82,13 +82,13 @@ function classify (s) {
  * @param options
  * @return {Object}
  */
-async function update (nameHash, target, options = {}) {
+async function update (nameId, target, options = {}) {
   const opt = R.merge(this.Ae.defaults, options)
-
   const nameUpdateTx = await this.nameUpdateTx(R.merge(opt, {
-    nameHash,
-    account: await this.address(),
-    pointers: JSON.stringify(R.fromPairs([[classify(target), target]]))
+    nameId: nameId,
+    accountId: await this.address(),
+    // pointers: R.fromPairs([[classify(target), target]])
```

Comment:
Looks like this can be removed.

Reply:
mh. no, this will not make `updates names` test pass...

Reply:
...or am I missing something?

Reply:
I mean this comment on line 90.

Reply:
okok, [done](https://github.com/aeternity/aepp-sdk-js/pull/79/commits/71549f330cd1464eac24b16240736633cc443d82)